eclipse-scout-core/src/tabbox/SimpleTab.js

```diff
@@ -209,26 +209,35 @@ export default class SimpleTab extends Widget {
   }
 
   _renderStatus() {
-    this._statusContainerUsageCounter -= (this._statusIconDivs.length === 0 ? 0 : 1);
+    this._statusContainerUsageCounter -= (this._statusIconSpans.length === 0 ? 0 : 1);
 
-    this._statusIconDivs.forEach($statusIcon => {
+    this._statusIconSpans.forEach($statusIcon => {
       $statusIcon.remove();
     });
-    this._statusIconDivs = [];
+    this._statusIconSpans = [];
 
     if (this.status) {
       this.status.asFlatList().forEach(status => {
-        if (!status || !status.iconId) {
+        if (!status || (!status.iconId && !status.message)) {
           return;
         }
-        let $statusIcon = this.$statusContainer.appendIcon(status.iconId, 'status');
-        if (status.cssClass()) {
-          $statusIcon.addClass(status.cssClass());
+        if (status.iconId) {
+          let $statusIcon = this.$statusContainer.appendIcon(status.iconId, 'status');
+          if (status.cssClass()) {
+            $statusIcon.addClass(status.cssClass());
+          }
+          this._statusIconSpans.push($statusIcon);
+        }
+        if (status.message) {
+          let $statusMessage = this.$statusContainer.appendSpan('status message', status.message);
```

Comment:
there is no css-class _message_ defined in SimpleTab.less

Reply:
Done 👍